io/io/inc/ROOT/RRawFile.hxx

```diff
@@ -50,6 +50,8 @@ public:
       ELineBreaks fLineBreak = ELineBreaks::kAuto;
       /// Read at least fBlockSize bytes at a time. A value of zero turns off I/O buffering. A negative value indicates
       /// that the protocol-dependent default block size should be used.
+      /// After construction, a negative block size is used to store the block size value when caching is turned off
+      /// (see `[Enable|Disable]Buffering()`).
       int fBlockSize = -1;
```

Comment:
Unrelated but `int` has an imprecise size (32bits or 64bits depending on plarform).  Does that matter?

Reply:
I think it is ok because variable contains the length of a memory buffer.

Reply:
And somewhat related to the PR, I wonder if using something like:
```suggestion
      Bool fCachingEnabled : 1 = true; 
      usigned int fBlockSize : 31 = 0;  // 0 indicates the protocol-dependent default block size should be used if caching is enabled.
```
would be clearer.

Reply:
> I think it is ok because variable contains the length of a memory buffer.

Then 'technically' it should be `size_t`, isn't?

Reply:
Yes, but because I'd like to use the negative value to, I'd need `ssize_t`, which is not part of C++.

Reply:
```std::make_signed<size_t>``` works in C++11 :)

Reply:
In related musing:
```
    size_t i : sizeof(size_t)-1;
```
works (initializing in place is a C++20 feature).